src/slash-commands/avatar.json

```diff
@@ -7,6 +7,22 @@
       "description": "The user to retrieve the avatar for, default you",
       "type": 6,
       "required": false
+    },
+    {
+      "name": "type",
+      "description": "The type of avatar to display",
```

Comment:
I like to add ", default server avatar" in the description for optional options, just so people know what is the default behavior